lib/beaker/host/unix/exec.rb

```diff
@@ -278,6 +278,8 @@ def clear_env_var key
   # @return [Result] result of restarting the SSH service
   def ssh_service_restart
     case self['platform']
+    when /ubuntu-24.04-/
```

Comment:
Since all of Debian (and its derivatives) use `ssh` instead of `sshd`, I'd prefer to make this:
```suggestion
    when /debian|ubuntu/
```

Just like it was prior to 3fb9a1bd857d21995d15d6cf876ce5c64fda0bb0.